File: sujet_machine/exemples/a_table/skeleton/a_table.cs
using System;
using System.Collections.Generic;

public class a_table
{
  static void a_table0(int n, int m, char[][] salle)
  {
    //  Inserez votre code ici 
  }
  
  public static void Main(String[] args)
  {
    int[] ba = new List<string>(Console.ReadLine().Split(" ".ToCharArray())).ConvertAll(int.Parse).ToArray();
    int n = ba[0];
    int m = ba[1];
    char[][] bb = new char[n][];
    for (int bc = 0; bc < n; bc++)
        bb[bc] = Console.ReadLine().ToCharArray();
    char[][] salle = bb;
    a_table0(n, m, salle);
  }
  
}

File: sujet_machine/exemples/a_table/skeleton/a_table.vb
Imports System
Imports System.Collections.Generic

Module a_table

  Sub a_table0(ByVal n as Integer, ByVal m as Integer, ByRef salle as Char()())
    ' Inserez votre code ici 
    
  End Sub
  Sub Main()
    Dim dm As Integer() = Array(Of String).ConvertAll(Of String, Integer)(Console.ReadLine().Split(" ".ToCharArray()), New Converter(Of String, Integer)(AddressOf Integer.Parse))
    Dim n As Integer = dm(0)
    Dim m As Integer = dm(1)
    Dim dn(n)() As Char
    For dp As Integer = 0 To n - 1
        dn(dp) = Console.ReadLine().ToCharArray()
    Next
    Dim salle As Char()() = dn
    a_table0(n, m, salle)
    End Sub
    
  End Module
  
  